### PR TITLE
Fix .scarymovie command (Halloween)

### LINKED ISSUE #769: TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
Sentry Issue: [SIR-LANCEBOT-5N](https://sentry.io/organizations/python-discord/issues/2440262025/?referrer=github_integration)

```
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
  File "discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "bot/exts/halloween/scarymovie.py", line 27, in random_movie
    selection = await self.select_movie()
  File "bot/exts/halloween/scarymovie.py", line 50, in select_movie
    params["page"] = random.randint(1, total_pages)
  File "random.py", line 339, in randint
    return self.randrange(a, b+1)

Unhandled command error: unsupported operand type(s) for +: 'NoneType' and 'int'
```

`total_pages` could be None if the data isn't included in the response. We should handle this.

### PR DESCRIPTION
**Pull request by D0rs4n**
## Relevant Issues

Closes #769 

## Description
I modified the code to use the proper API endpoint, and changed the HTTP parameters accordingly.
Also, I removed a reference to an environment variable, that was unnecessary. ( That parameter (TMDB_TOKEN) wasn't even present in the list on the [Wiki](https://pythondiscord.com/pages/guides/pydis-guides/contributing/sir-lancebot/env-var-reference/)).
I have also added a new filter `"include_adult": "false"` , to make sure it won't return inappropriate results.
(That was my first ever pull request to an open-source project, I'd like to apologize for the possible mistakes I made, in advance) 
## Did you:
- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

**Review by ToxicKidz — COMMENTED** [review]
Looks good to me, one comment

File: bot/exts/halloween/scarymovie.py [hunk 1]
```diff
@@ -11,7 +11,6 @@
 
 
 TMDB_API_KEY = environ.get("TMDB_API_KEY")
```
[the remark this answers is not in the document]
Comment (D0rs4n) [reply bot/exts/halloween/scarymovie.py:13]: Sure, I'll make a commit shortly.

**Review by ToxicKidz — APPROVED** [review]

**Review by JacobMonck — APPROVED** [review]
Looks good to me! 🎉

**Review by jchristgit — APPROVED** [review]
Rock solid.

**Review by Xithrius — APPROVED** [review]

**Comment by Xithrius** [comment]
@D0rs4n Thank you for the PR!

**Pull request merged**